tests/test_formatters.py

```diff
@@ -324,6 +324,25 @@ def test_level_2_header(markdown_function, expected):
     assert markdown_function('## inset text') == expected
 
 
+@pytest.mark.parametrize('markdown_function, expected', (
+    [
+        notify_email_markdown,
+        '<h3 style="Margin: 0 0 15px 0; padding: 0; line-height: 26px; color: #323A45;'
+        'font-size: 20.8px; font-weight: bold; font-family: Helvetica, Arial, sans-serif;">inset text</h3>'
+    ],
+    [
+        notify_plain_text_email_markdown,
+        (
+            '\n'
+            '\ninset text'
+            '\n-----------------------------------------------------------------'
+        ),
+    ],
+))
+def test_level_3_header(markdown_function, expected):
```

Comment:
this needs to follow the test naming convention.

Reply:
Existing unit tests are named `test_level_1_header` and `test_level_2_header`.  Is it better to give a new test an inconsistent name?

Reply:
Kyle approved the test as it stands.

Reply:
Logic:
Testing guide is kind of vague [for naming](https://github.com/department-of-veterans-affairs/vanotify-team/blob/master/Process/testing_guide.md#test-naming).
> Naming shall be consistent

Could be interpreted as continuing the existing naming convention. In the case of that PR it'd be weird to name it different when it's a copy/paste of the other two. We didn't want to rename existing tests under most circumstances. I think maintaining the name is fine in this case.